### ISSUE TITLE
Script for ASR decoding?

### ISSUE THREAD

**Issue by manish-kumar-garg** [tensorflow/lingvo#163]
I am training the grapheme model for ASR training.
45K steps are done. 
I want to calculate the WER on train, dev and test set. How should I do it?
How should I decode using the model checkpoints?

**Comment by manish-kumar-garg** [comment]
@drpngx

**Comment by drpngx** [comment]
Are you looking at the tensorboard? It's reported there. Otherwise, there is a script in [tools](https://github.com/tensorflow/lingvo/tree/master/lingvo/tasks/asr/tools) called `simple_wer.py`.

I think this is a dup of #42

**Comment by manish-kumar-garg** [comment]
Thanks
